Sds trustworthy jwt doc (#4701)
istio/istio.io: content/en/docs/setup/platform-setup/minikube/index.md

Follow these instructions to prepare minikube for Istio installation with sufficient
resources to run Istio and some basic applications.

## Prerequisites

- Administrative privileges are required to run minikube.

> - To enable SDS in Istio, add [extra configurations](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) to your Kubernetes. You may

Reviewer: ```suggestion
- To enable SDS for your mesh, you must add [extra configurations](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) to your Kubernetes deployment.
```

> also want to refer to the [api-server page](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) for the most up-to-date flag names.

Reviewer: ```suggestion
Refer to the [`api-server` reference docs](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) for the most up-to-date flags.
```
